src/pages/index.html

```diff
@@ -6,11 +6,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calculator </title>
-    <link rel="stylesheet" href="css11.css">
+    <link rel="stylesheet" href="/src/assets/Main.css">
```

Comment:
it should be from dist folder, shouldn't it?

Reply:
> it should be from dist folder, shouldn't it?

aah that's true, the dist it's suppose to have all the assets file?
like js and css

Reply:
yep